config/environments/production.rb

```diff
@@ -33,11 +33,12 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for nginx
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  #config.force_ssl = true
+  # config.force_ssl = true
   # config.middleware.insert_before ActionDispatch::Static, Rack::SSL, :exclude => proc { |env| env['HTTPS'] != 'on' }
 
-  # See everything in the log (default is :info)
-  # config.log_level = :debug
+  # Set to `:info` to match the current default, or set to `:debug` to opt-into
+  # the future default.
+  config.log_level = :info
```

Comment:
Same as above

Reply:
This was added by @jywarren in the update process